From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Muhammad Alfi Syahrin <malfisya.dev@hotmail.com>
Date: Mon, 16 Feb 2026 09:27:35 +0700
Subject: [PATCH] Rename Krokiet app icon

Krokiet icon listed in `io.github.qarmin.krokiet.desktop` as
io.github.qarmin.krokiet. Make the the actual app icon file and the one
listed in the desktop file the same.
---
 ...ub.qarmin.czkawka.krokiet.svg => io.github.qarmin.krokiet.svg} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename data/icons/{io.github.qarmin.czkawka.krokiet.svg => io.github.qarmin.krokiet.svg} (100%)

diff --git a/data/icons/io.github.qarmin.czkawka.krokiet.svg b/data/icons/io.github.qarmin.krokiet.svg
similarity index 100%
rename from data/icons/io.github.qarmin.czkawka.krokiet.svg
rename to data/icons/io.github.qarmin.krokiet.svg
